phalcon/Mvc/Model/Query/Builder.zep

```diff
@@ -633,17 +638,7 @@ class Builder implements BuilderInterface, InjectionAwareInterface
              * If the conditions is a single numeric field. We internally create
              * a condition using the related primary key
              */
-            if typeof models == "array" {
-                if unlikely count(models) > 1 {
-                    throw new Exception(
-                        "Cannot build the query. Invalid condition"
-                    );
-                }
```

Comment:
Could you please explain this removal in the PR description and mention this in the change log

Reply:
It already make same check above

https://github.com/phalcon/cphalcon/pull/14758/files#diff-6801ddb1a4b13b44acd35b479d000f4bR620-R625

Reply:
No.
```diff
-   if unlikely !count(models)
+   if unlikely count(models) > 1
```

Reply:
I doubt if this really needed here, but will revert anyway.